build-jenkins-agent-windows.pkr.hcl

```diff
@@ -81,14 +75,6 @@ build {
     script            = "./provisioning/windows-provision.ps1"
   }
 
-  # Recommended (and sometimes required) before running deprovisioning (sysprep)
-  # ref. https:#www.packer.io/docs/builders/azure/arm#windows
-  provisioner "windows-restart" {
-    max_retries = 3
-    # Previous provisioner might restart
-    pause_before = "1m"
-  }
```

Comment:
Interesting: this "reboot" removal make the build stuck in the sysprep phase but only in AWS:

```
Info: Sysprep command: C:\Windows\System32\Sysprep\Sysprep.exe /oobe /quit /generalize /unattend:C:\ProgramData\Amazon\EC2Launch\sysprep\unattend.xml
```